exchange/exchange.go

```diff
@@ -218,7 +221,7 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 	var liveAdapters []openrtb_ext.BidderName
 
 	if len(r.StoredAuctionResponses) > 0 {
-		adapterBids, liveAdapters, err = buildStoredAuctionResponse(r.StoredAuctionResponses)
+		adapterBids, liveAdapters, err = buildStoreAuctionResponse(r.StoredAuctionResponses)
```

Comment:
This might be a spelling error, do we want this to say `buildStoredAuctionResponse` instead of `buildStoreAuctionResponse`

Reply:
Renamed. It was fixed in Stored Single Response PR, but I rebased it incorrect. Good catch!